### PR TITLE
feat: Implement redux storage

### PR DESCRIPTION
**Pull request by Pedrael**

### CODE REVIEW

File: src/pages/products/index.page.tsx [hunk 1]
```diff
@@ -1,7 +1,19 @@
+import { useSelector } from 'react-redux'
+import { State } from '../../types'
+
 export const Page = () => {
+  const orders = useSelector((state: State) => state.orders)
   return (
     <>
       <h1>Products</h1>
+      <div>
+        {orders.map(({ title, date, description, products }, id) => (
```
Comment (ReidenXerx) [inline src/pages/products/index.page.tsx]: No sequenced number as key

File: src/store.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        { value: 100, symbol: Currency.USD, isDefault: 0 },
+        { value: 2600, symbol: Currency.UAH, isDefault: 1 },
+      ],
+      date: new Date('2017-06-29 12:09:33'),
+    },
+  ],
+}
+
+const rootReducer = (state = initialState, { payload, type }: Action) => {
+  switch (type) {
```
Comment (ReidenXerx) [inline src/store.ts]: No switch cases
Use hash handler object

**Review by ReidenXerx — CHANGES_REQUESTED** [review]

File: src/pages/products/index.page.tsx (hunk 1 above)
Comment (Pedrael) [reply src/pages/products/index.page.tsx]: fixed

File: src/store.ts (hunk 2 above)
Comment (Pedrael) [reply src/store.ts]: fixed

File: src/components/Navmenu.tsx [hunk 3]
```diff
@@ -3,11 +3,16 @@ import { Link } from '../renderer/Link'
 import { useTranslation } from 'react-i18next'
 import { Label } from './Label'
 import { cream, red, Paths } from '../constants'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 
 export const Navmenu = () => {
-  const [url, setUrl] = useState<string>('/')
+  const [url, setUrl] = useState<string | undefined>(undefined)
```
Comment (ReidenXerx) [inline src/components/Navmenu.tsx]: Why path initially is undefined?

Comment (Pedrael) [reply src/components/Navmenu.tsx]: because I get if from the URL. If undefined it is not rendered

Comment (Pedrael) [reply src/components/Navmenu.tsx]: fixed

**Review by ReidenXerx — APPROVED** [review]

**Pull request merged**